data/mappings/info_config.json

```diff
@@ -61,6 +61,7 @@
     "ONESHOT_TAP_TOGGLE": {"info_key": "oneshot.tap_toggle", "value_type": "int"},
     "PERMISSIVE_HOLD": {"info_key": "tapping.permissive_hold", "value_type": "bool"},
     "PERMISSIVE_HOLD_PER_KEY": {"info_key": "tapping.permissive_hold_per_key", "value_type": "bool"},
+    "PRESSED_KEY_PIN_STATE": {"info_key": "pressed_key.pin_state", "value_type": "int"},
```

Comment:
Also the C-level option should probably be renamed to something like `MATRIX_INPUT_PRESSED_STATE` too.

Reply:
Wasn't sure about the JSON placement, so I modeled this after `indicators.on_state` to keep changes minimal. `matrix_pins.input_pressed_state` sounds like a better name and placement indeed. Messing with `output_active_state` would be the next step, but would require some heavy rework of the matrix logic and is out of the scope of this PR in terms of functionality.
This also opens the discussion if `readMatrixPin(pin)` needs to become a boolean function since it's returning only 1/0, which I avoided due to the bitwise operations going on in hardware scanning routines. It will probably still work as boolean, but would the code be consistent as is? Please let me know if this is desirable